### DISCUSSION TITLE
Evolution Theme not showing "As low as" price in category view.

### DISCUSSION THREAD

**Discussion by pinkstond** [breezefront/community#171] [1. Help] [answered]
I'm currently using the Breeze Evolution theme. I'm wanting to show products in the category grid view the same way they are displayed in the Breeze Blank theme.  Specifically, I'd like the "As low as" price to show. But it'd be nice to have the "Add to Cart" button at the bottom rather than as a hover over the photo. 

Current category grid view: 
![image](https://github.com/breezefront/community/assets/58951887/77ca0aba-9c34-4126-a546-0acec5bda272)
Desired category grid view: 
![image](https://github.com/breezefront/community/assets/58951887/2a32aac2-6308-49b4-853c-8cb8672f952a)

I feel like I've searched and edited every possible file in the vendor/swissup/theme-frontend-breeze-evolution directories and nothing seem to be getting close to achieving the results I'm looking for.  Any guidance would be very much appreciated! 

Guidance resulting in desired results, next beer (or drink of choice) is on me! :)

**Comment by vovayatsyuk** [comment, marked as answer]
It's controlled via less [_variables.less](https://github.com/breezefront/theme-frontend-breeze-evolution/blob/master/web/css/theme/abstracts/_variables.less):

```
@price-normal-label__display: none;
@price-old-label__display: none;
@listing-grid-item-actions-primary__position: absolute;
```

See the default values in https://breezefront.com/docs/listing and https://breezefront.com/docs/price

Maybe you'll also want to show the prices below the product name:

```
@listing-grid-item__price-aside-name: false;
```

:beers:

**Reply by pinkstond** [reply]
[vovayatsyuk](https://github.com/vovayatsyuk), thank you for your response!

I have changed and experimented with these variables many times the past few days. The specific one I've been editing is found here: 
vendor/swissup/theme-frontend-breeze-evolution/web/css/theme/abstracts/_variables.less

After making the changes I have tried the following from command line: 
deploying static-content
flushing the cache
cleaning the cache
compiling
reindexing

After doing so, the changes do not appear to take place for some reason and it has had me scratching my head.

**Reply by vovayatsyuk** [reply]
Just a note: Please do not edit evolution/blank theme files. Always create a child theme for all changes.

https://breezefront.com/docs/child-theme

Or use `bin/magento breeze:theme:create` command.

**Comment by pinkstond** [comment]
Got it. It appears to have accepted the changes after manually clearing /pub/static & /var/view_preprocessed and then deploying static-content and flushing/cleaning cache.

Thanks for your help!
